utils/src/main/java/io/helidon/build/util/Constants.java

```diff
@@ -31,14 +31,15 @@
     public static final OSType OS = currentOS();
 
     /**
-     * End of line string.
+     * End of line string.                                                                                                                                                                        `
```

Comment:
Looks like that character was not intended.

Reply:
Ah, invisible control character; fixed.